src/InventoryList/Cart.pm

```diff
@@ -0,0 +1,56 @@
+#########################################################################
+#  OpenKore - Player actor object
+#  Copyright (c) 2005 OpenKore Team
+#
+#	TEST BY HENRYBK
+#
+#########################################################################
+package InventoryList::Cart;
+
+use strict;
+use Globals;
+use InventoryList;
+use base qw(InventoryList);
+
+sub new {
+	my ($class) = @_;
+	my $self = $class->SUPER::new('Cart');
```

Comment:
I don't think passing a parameter to `InventoryList->new` does anything. Might as well remove it? `my $self = $class->SUPER::new;`